tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -109,7 +108,7 @@ def test_check_sfputil_reset(duthosts, enum_rand_one_per_hwsku_frontend_hostname
             assert reset_result["rc"] == 0, "'{} {}' failed".format(cmd_sfp_reset, intf)
             time.sleep(5)
     logging.info("Wait some time for SFP to fully recover after reset")
-    time.sleep(60)
+    time.sleep(sleep_time)
```

Comment:
is sleep_time defined in this functions?

Reply:
Some codes were missing.  I pushed a fix.